modules/images-s2i-nodejs-pulling-images.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-s2i-nodejs-pulling-images_{context}"]
+= Pulling images for Node.js
+
+These images come in two options:
+
+* RHEL 8
+* CentOS 7
+
+*RHEL 8 Based Images*
+
+The RHEL 8 images are available through the Red Hat Registry.
+
+.Procedure
+
+* To pull the RHEL 8 image, enter the following command for the version of Node.js you want:
++
+.Node.js `0.10`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/openshift4/nodejs-010-rhel7
+----
++
+.Node.js `4`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/nodejs-4-rhel8
+----
+
+*CentOS 7 Based Image*
+
+This image is available on link:quay.io[Quay.io].
+
+.Procedure
+
+* To pull the CentOS 7 image, enter the following command:
++
+[source,terminal]
+----
+$ podman pull openshift/nodejs-010-centos7
```

Comment:
Above three images are unavailable.
$skopeo inspect docker://registry.redhat.io/openshift4/nodejs-010-rhel7
FATA[0001] unknown: Not Found 
$skopeo inspect docker://registry.redhat.io/rhscl/nodejs-4-rhel8
FATA[0001] unknown: Not Found

openshift/nodejs-010-centos7 is pulling limited by docker hub pull limit

We can use registry.redhat.io/rhscl/nodejs-12-rhel7:latest and registry.redhat.io/rhscl/nodejs-10-rhel7:latest

Reply:
Thank you! I'm commenting out the references to Centos for now, but can remove them entirely if it's no longer available in 4.x.